content/docs/overview.md

```diff
@@ -3807,6 +3843,36 @@ n := halve(4).? or_else 0
 fmt.println(n)                   // 2
 ```
 
+#### Testing
+
+Odin includes [a "core:testing" package](https://pkg.odin-lang.org/core/testing/) for unit testing purposes.
+
+A minimalist example test:
+
+```odin
+// tests/test_code.odin
+
+package test_code
+
+import "core:testing"
+
+@test
+test_that_hello_world_equals :: proc(t: ^testing.T) {
+	expected := "Hello, World!"
+
+	// More likely this is a call to a function you want to test
+	result := "Goodbye, Mars!"
+
+	testing.expect_value(t, result, expected)
+}
+```
+
+You can run this test using `odin test tests/`. Any tests in the given directory that are annotated with `@test` will be run. You can also run a single test file using `odin test tests/test_code.odin -file`.
+
+**Note:** Odin files ending in `_test.odin` are only compiled when building/running tests with `odin test`. Your test files are not required to have this filename suffix to run with `odin test`, but you can use it to prevent code from building included during normal `odin build` or `odin run` commands. This filename suffix is similar to `_windows.odin` files, which will only be compiled if the target OS is Windows.
```

Comment:
I might replace this paragraph with something like:

> You may also use the `test` [file suffix](https://odin-lang.org/docs/overview/#file-suffixes) to only include a file in the build when testing. e.g: `foo_test.odin`.
> Just like other file suffixes, this means that the file will not even be analysed unless you are running `odin test`.

Reply:
Good idea; done!